src/Pages/Nodes.razor

```diff
@@ -319,36 +385,71 @@
                     _nodes = await NodeRepository.GetAll();
 
                 }
-
+                await StopJob(node);
             }
         }
 
         await GetData();
 
     }
 
+
+    private async Task StopJob(Node node)
+    {
+        try
+        {
+            IScheduler scheduler = await SchedulerFactory.GetScheduler();
+            
+            await SimpleJob.UnscheduleJob<NodeChannelSuscribeJob>(scheduler, node.Id.ToString());
+            await SimpleJob.UnscheduleJob<ProcessNodeChannelAcceptorJob>(scheduler, node.Id.ToString());
+            
+            node.JobId = null;
+            _NodesState(node);
+        }
+        catch
+        {
+            ToastService.ShowError("Error while stopping jobs, please contact a superadmin for troubleshooting");
+        }
+    }
+
     private async Task CreateJobs(Node node)
     {
         try
         {
             IScheduler scheduler = await SchedulerFactory.GetScheduler();
 
             var map = new JobDataMap();
-            map.Put("nodeId", node.Id);
-
-            var job = SimpleJob.Create<NodeChannelSuscribeJob>(map, node.Id.ToString());
-            await scheduler.ScheduleJob(job.Job, job.Trigger);
-
-            node.JobId = job.Job.Key.ToString();
+            var isJobExists = await SimpleJob.IsJobExists<NodeChannelSuscribeJob>(scheduler, node.Id.ToString());
```

Comment:
Maybe you can create a function `Upsert` instead to avoid so much repetition?

Reply:
Ok, could be improved (DRY)